### ISSUE TITLE
velaux  1.9.1  not supported multiple gateway traits on a component

### ISSUE THREAD

**Issue by Louse-luo** [kubevela/velaux#859]
**Describe the bug**

velaux  1.9.1  not supported multiple gateway traits on a component
**To Reproduce**

Configuring the second gateway trait through velaux, reporting an error
![image](https://github.com/kubevela/velaux/assets/13954320/daf65ce8-4673-402a-8642-9e4d0d26f831)

**Expected behavior**

**Screenshots**

**KubeVela Version**
velaux  1.9.1
CLI Version: 1.9.4
Core Version: 1.9.4
GitRevision: git-b9f1cc97
GolangVersion: go1.19.10

**Additional context**

**Comment by chivalryq** [comment]
dup https://github.com/kubevela/velaux/issues/829

**Comment by lxkaka** [comment]
@Louse-luo if you need multi domains for ingress, you can customize gateway

**Comment by wonderflow** [comment]
yes, this is an issue of veluax as kubevela already supported this feature.
